site/content/entry/yuzu-progress-report-jul-2022/index.md

```diff
@@ -0,0 +1,483 @@
+Remaining relevant hardware used is a 5600X and 16GB of RAM at 3600MHz, the RX 6600 was running at PCIe 4.0 8x with Smart Access Memory enabled, although that won’t make a difference, more on that later.
+Operating systems used are Windows 11 and Manjaro Linux, both up to date and on their respective default stable branches.
+yuzu is on Mainline 1112, with GPU accuracy set to normal to make GPU driver bottlenecks easier to measure, 1X resolution multiplier and Default value for Anisotropic Filtering.
+
+A single regression under investigation and reported to AMD aside (Xenoblade Chronicles 2 crashes loading Abble’s Fountain, the measuring spot, could be caused by some driver thread crash), performance is now very close to Vulkan numbers, be it either from AMD or Nvidia. 
+
+It’s now perfectly valid for a Radeon user to switch to OpenGL if a specific game requires it, like for example with Xenoblade Chronicles 3, or an Unity/Unreal Engine based game (SHIN MEGAMI TENSEI V).
+As a bonus, while not being very stable, the SPIR-V shader back-end can be used on games with “simple” shaders like Super Smash Bros. Ultimate or Super Mario Odyssey, making shader building much more tolerable when compared to GLSL, giving it a performance much closer to the Nvidia-only GLASM.
+
+Another lesson learned from this is that some games, like Legend of Zelda: Breath of the Wild, just outright prefer Nvidia’s mature OpenGL driver. Ara ara.
+
+Lastly, to end this Red Team section.
+[In the past](https://yuzu-emu.org/entry/yuzu-progress-report-feb-2022/#vulkan-is-the-future) we reported a way to defeat RDNA2’s overcorrecting power manager in order to get decent framerates.
+This method, while simple, has a downside: It’s an overclock.
+Or at least counts as one.
+
+We found an alternative that should be more globally applicable.
+The trick this time is to make the driver force high clocks on a more important section of the GPU when speaking about emulation performance in general: GPU VRAM.
+All this while keeping the warranty in check.
+
+The process is simple, make the integrated video encoder work in the background while yuzu (or any other emulator) runs.
+This is easily achieved from Radeon Software by going to `Settings > Record & Stream` and enabling `Instant Replay`.
+Intel/Linux owners should be able to reach similar results by instead using the Xbox Game Bar, or [setting OBS to keep a buffer](https://obsproject.com/forum/resources/instant-replay.575/).
+
+{{< imgs
+	"./instantreplay.png| A driver toggle for power management would be so much simpler"
+  >}}
+
+After this, on yuzu enable Exclusive Fullscreen from `Emulation > Configure.. > Graphics > Fullscreen Mode`.
+Then just play your games in fullscreen by pressing F11.
+This step can be avoided if you also enable `Record Desktop`, but keep in mind this would increase your power consumption even while doing nothing.
```

Comment:
```suggestion
After this, in yuzu enable Exclusive Fullscreen from `Emulation > Configure.. > Graphics > Fullscreen Mode`.
Then just play your games in fullscreen by pressing F11.
This step can be avoided if you also enable `Record Desktop`, but please keep in mind this will increase your power consumption even while idling.
```